ui/src/components/NewExperimentNext/data/types.ts

```diff
@@ -534,14 +541,430 @@ const data: Record<Kind, Definition> = {
   },
 }
 
-const AwsChaosCommonSchema = Yup.object({
+const networkPhysicCommon: Spec = {
+  correlation: {
+    field: 'text',
+    label: 'Correlation',
+    value: '',
+    helperText: 'The correlation of corrupt',
+  },
+  device: {
```

Comment:
@WangXiangUSTC `device` is a required field, it needs to be validated by webhook.